torch_xla/experimental/plugins.py

```diff
@@ -65,3 +74,18 @@ def default() -> DevicePlugin:
 def register_plugin(name: str, device_plugin: DevicePlugin):
   _plugin_registry[name.upper()] = device_plugin
   torch_xla._XLAC._register_pjrt_plugin(name, device_plugin.library_path())
+
+
+def register_installed_plugins():
+  pjrt_entry_points = importlib_metadata.entry_points(group='torch_xla.plugins')
```

Comment:
I wonder what this line does under the hood

Reply:
Entry points are weird. [`setuptools`' docs](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins) have the best explanation that I've read.